src/converters.ts

```diff
@@ -117,11 +137,8 @@ class Decimal implements IConverter<string, string> {
   }
 }
 
-function decimal(
-  maxDigits: number,
-  decimalPlaces: number
-): IConverter<string, string> {
-  return new Decimal(maxDigits, decimalPlaces);
+function decimal(props: any): IConverter<string, string> {
```

Comment:
use DecimalProps here instead of any. This way people's editors will help them show what options are available.